docs/src/pages/quick-start.md

```diff
@@ -3,169 +3,56 @@ id: quick-start
 
-## Configuration file
+## Run Monika
 
-> The configuration file contains the [notification](/monika/guides/notifications), [probes](/monika/guides/probes), and [alerts](/monika/guides/alerts) as shown below.
+Monika by default reads a configuration file called `monika.json` in the current working directory if it exists. To simply run previously downloaded the configuration, run this command on the Terminal in the same path you saved `monika.json` file:
 
-```
-// monika.json
-
-{
-  "notifications": [...],
-  "probes": [
-    {
-      ...
-      "alerts": [...]
-    }
-  ]
-}
+```bash
+monika
 ```
 
-For advanced configuration, you can find them on the sidebar menu.
-
-Monika by default reads a configuration file called `monika.json` in the current working directory if it exists. You can specify a path to a JSON configuration file with `-c` flag as follows
+You can specify a path to a JSON configuration file with `-c` flag if you didn't rename your configuration file as monika.json, as follows
 
 ```bash
 monika -c <path_to_configuration_json_file>
 ```
 
-Or if you haven't installed Monika globally, you can run it without installing first using [npx](https://www.npmjs.com/package/npx):
+Better yet, you can provide an URL that contains monika configuration
 
 ```bash
-npx @hyperjumptech/monika -c <path_to_configuration_json_file>
+monika -c https://domain.com/path/to/your/configuration.json
 ```
 
-## Background run
-
-By default Monika will run in the foreground. Like other Node.js applications, there are several ways to run Monika in the background on Unix, Linux, and macOS.
-
-### Using `nohup`
-
-- On your terminal, run `nohup monika &`
-- You'll get an output similar to the following.
-
-  ```
-  [1] 93457
-  appending output to nohup.out
-  ```
-
-  In the above example, 93457 is the process ID (pid). And the output of Monika is written to `nohup.out` file.
-
-- To stop Monika, run `kill -9 <pid>`.
-
-### Using `screen`
-
-- On Debian/Ubuntu, you can install it by running `sudo apt install screen`.
-- Run `screen`.
-- Run `monika -c monika.json`
-- Press Ctrl+a then D. This will cause Monika to run on a different screen in the background.
-- To go back to the screen, run `screen -ls` to list the running screens. You will get an output similar to the following.
-
-  ```
-  There is a screen on:
-    9049.pts-0.the-server	(03/23/21 08:34:38)	(Detached)
-    1 Socket in /run/screen/S-server.
-  ```
+Congratulations, you have successfully run Monika in your machine!
 
-  `9049.pts-0.the-server` is the name of the screen.
+## Next Step
 
-- Then run `screen -r <name_of_the_screen>`.
-- To stop Monika, hit Ctrl+c then Ctrl+d.
+For advanced configuration such as configuring notifications, probes, and alerts, you can find them on the sidebar menu for more details.
```

Comment:
`For advanced configuration such as configuring notifications, probes, and alerts, you can find them on the sidebar menu.`